src/internal/components/pluggableVisualizations/xirr/xirrBucketHelper.ts

```diff
@@ -0,0 +1,37 @@
+// (C) 2019 GoodData Corporation
+import flatMap = require("lodash/flatMap");
+import { IReferencePoint, IBucket, IBucketItem } from "../../../interfaces/Visualization";
+import { BucketNames } from "../../../..";
+
+const getMeasureBucket = (buckets: IBucket[]): IBucket | undefined =>
```

Comment:
try to use generic bucket helpers as we discussed on Slack

Reply:
Changed, thank you